src/main/java/oscar/form/study/HSFO/pageUtil/HsfoQuartzServlet.java

```diff
@@ -37,10 +37,7 @@
 
 import org.apache.logging.log4j.Logger;
 import org.oscarehr.util.MiscUtils;
-import org.quartz.JobDetail;
-import org.quartz.Scheduler;
-import org.quartz.SimpleTrigger;
-import org.quartz.Trigger;
+import org.quartz.*;
```

Comment:
best practice is to avoid wildcard imports - can we just list the needed ones?

Reply:
Done